app/admin/user.rb

```diff
@@ -1,12 +1,15 @@
 ActiveAdmin.register User do
   filter :email
+  permit_params [:email, :admin, :password, :password_confirmation]
 
   form do |f|
-    inputs 'Details' do
-      input :email
-      input :admin
-      actions 
+    f.inputs 'Details' do
+      f.input :email
+      f.input :admin
+      f.input :password
+      f.input :password_confirmation
     end
+    f.actions
```

Comment:
Cool.

ActiveAdmin has you use this single `form` method to compose both the "create" and the "edit" forms. Although similar, when creating a new user you'll want to specify a `password` / `password_confirmation` but it shall not be modified in the edit form, so you'd like to hide/remove those two inputs.

You can check whether the form is in "create" or "edit" mode by checking whether the `User` resource that the form handles is an existing record or not: `f.object.new_record?` (where `f.object` is a reference to the `User` instance being either created or modified). There is an example in [the documentation about the *Customizing the Form*](http://activeadmin.info/docs/5-forms.html).